applications/GeoMechanicsApplication/tests/test_transient_thermal.py

```diff
@@ -20,424 +21,189 @@ def setUp(self):
     def tearDown(self):
         # Code here will be placed AFTER every test in this TestCase.
         pass
-
-    def test_thermal_heat_flux_2D3N(self):
-        test_name = 'test_thermal_heat_flux_2D3N'
-        file_path = test_helper.get_file_path(os.path.join('test_thermal_element', 'test_thermal_heat_flux', test_name))
+        
+    def check_temperature(self, test_directory, test_name, test_node, etalon_value):
```

Comment:
To me, from the name it's not directly clear that this function actually runs a simulation before checking results. Do you think we should reflect that in the name?

Reply:
Indeed, I do agree that the name of the test must reflects exactly what the test is meant for. Do you have suggestion for a more clear name?

Reply:
We could consider to move the check itself back to the test bodies, and only run the thermal analysis here, which returns a list of temperatures. In the test body, all you need to do extra is get the temperature value at the relevant node (from the returned list of temperatures) and compare it to an expected value. That would reduce the parameter list here (since you no longer need `test_node` and `etalon_value`), and it would make it simpler to give this function a proper name, e.g. `run_thermal_analysis`. Each test then consists of two lines: (1) running the thermal analysis, and (2) comparing the relevant temperature value. What do you think?